packages/space-opera/src/components/model_viewer_preview/gltf_edits.ts

```diff
@@ -100,7 +100,9 @@ function dispatchGltf(args?: DispatchGltfArgs) {
   if (getEdits(reduxStore.getState()) === edits) {
     throw new Error(`Same edits was given! Only call this upon actual change`);
   }
-  reduxStore.dispatch(dispatchSetEdits(edits));
+  if (!isFromPoster) {
```

Comment:
we don't want to reset the edits if we are displaying a poster after material edits have been done

Reply:
"FromPoster" seems a little odd. I think what you mean is we're resetting the src to force the poster to be shown again, rather than loading a different src. Could we simplify this a bunch by changing model-viewer a little? Maybe show the poster when the poster is changed, or make a showPoster function? This reloading the model stuff seems like a hack anyway.

Reply:
there isn't really any "unload" method though, so I'm not sure exactly how else to do pull off a showPoster function?